tests/main/firstboot/task.yaml

```diff
@@ -1,68 +0,0 @@
-summary: Check that firstboot assertions are imported and snaps installed
```

Comment:
as I wrote in the commit for the removing, now we have classic-firstboot and various ubuntu-core- tests also check aspects of these,  this was written by you I think while working on prepare-image before we even had any ubuntu-core- tests